tensorflow/lite/micro/kernels/test_data_generation/generate_flexbuffers_data.cc

```diff
@@ -52,18 +58,28 @@ void generate(const char* name, bool use_regular_nms) {
   int fbb_size = fbb.GetBuffer().size();
 
   printf("const int g_gen_data_size_%s = %d;\n", name, fbb_size);
-  printf("const unsigned char g_gen_data_%s[] = { ", name);
+  printf("const unsigned char g_gen_data_%s[] = {\n", name);
+  print_spaces(4);
+  int count_newlines = 0;
   for (size_t i = 0; i < fbb_size; i++) {
-    printf("0x%02x, ", init_data[i]);
+    printf("0x%02x,", init_data[i]);
+    count_newlines++;
+    if ((count_newlines % 12) == 0) {
+      printf("\n");
+      print_spaces(4);
+    }
+    else if (i < (fbb_size-1)) {
+      print_spaces(1);
+    }
```

Comment:
I think we can skip this formatting code. Its easier to clang-format the output file.